app/lib/actions/pulp/consumer/abstract_content_action.rb

```diff
@@ -13,6 +13,21 @@ def external_task=(external_task_data)
           end
         end
 
+        def process_timeout
+          accept_timeout = ::Katello.config.content_action_accept_timeout
+          finish_timeout = ::Katello.config.content_action_finish_timeout
+          pulp_state = output[:pulp_tasks][0][:state]
+
+          if pulp_state == 'waiting'
+            fail _("Host did not respond within %s seconds.  Is katello-agent installed and goferd running on the Host?") % accept_timeout
+          elsif output[:client_accepted].nil?
+            output[:client_accepted] = Time.now.to_s
+            schedule_timeout(finish_timeout)
+          elsif pulp_state == 'running' || pulp_state == 'waiting'
```

Comment:
If the pulp_state is `waiting` the first condition will catch and never reach this?